pkgs/development/python-modules/atsim_potentials/default.nix

```diff
@@ -1,49 +1,62 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
 , configparser
 , pyparsing
-, pytest
+, pytestCheckHook
 , future
 , openpyxl
 , wrapt
+, scipy
+, cexprtk
+, deepdiff
+, sympy
 }:
 
 buildPythonPackage rec {
   version = "0.4.0";
   pname = "atsim.potentials";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0c3e4e2323e969880f17a9924642e0991be5761f50b254bcbadd046db3be6390";
+  src = fetchFromGitHub {
+    owner = "mjdrushton";
+    repo = "atsim-potentials";
+    rev = version;
+    sha256 = "sha256-MwjRVd54qa8uJOi9yRXU+Vrve50ndftJUl+TFZKVzQM=";
   };
 
-  checkInputs = [ pytest ];
+  postPatch = ''
+    # Remove conflicting openpyxl dependency version check
+    sed -i '10d' setup.py
```

Comment:
That is very brittle. Please use sed with the package name or better substituteInPlace